docs/sections/user_guide/cli/mode_template.rst

```diff
@@ -223,3 +225,74 @@ and YAML file ``values.yaml`` with content
      Hello, World!
 
   Note that ``ini`` and ``nml`` configs are, by definition, depth-2 configs, while ``sh`` configs are depth-1 and ``yaml`` configs have arbitrary depth.
+
+.. _cli_template_translate_examples:
+
+``translate``
+-------------
+
+.. code-block:: text
+
+   $ uw template translate --help
+   usage: uw template translate [-h] [--input-file PATH] [--output-file PATH] [--dry-run] [--quiet]
+                                [--verbose]
+
+   Translate atparse to Jinja2
+
+   Optional arguments:
+     -h, --help
+         Show help and exit
+     --input-file PATH, -i PATH
+         Path to input file (defaults to stdin)
+     --output-file PATH, -o PATH
+         Path to output file (defaults to stdout)
+     --dry-run
+         Only log info, making no changes
+     --quiet, -q
+         Print no logging messages
+     --verbose, -v
+         Print all logging messages
+
+Examples
+^^^^^^^^
+
+The examples that follow use atparse-formatted template file ``atparse.txt`` with content
+
+.. code-block:: text
+
+   @[greeting], @[recipient]!
+
+* Convert an atparse-formatted template file to Jinja2 format:
+
+  .. code-block:: text
+
+     $ uw template translate --input-file atparse.txt
+     {{greeting}}, {{recipient}}!
```

Comment:
Here and below, note that `--input-format` and `--output-format` are no longer used.